app/views/browse/replica-set.html

```diff
@@ -13,12 +13,11 @@
               <h1>
                 {{replicaSet.metadata.name}}
                 <span
-                  ng-if="deploymentConfigMissing"
+                  ng-if="deploymentMissing || deploymentConfigMissing"
                   class="pficon pficon-warning-triangle-o"
-                  style="cursor: help; vertical-align: middle;"
                   data-toggle="tooltip"
                   data-trigger="hover"
-                  title="The deployment's deployment config is missing."
+                  data-original-title="{{deploymentConfigMissing ? 'The deployment configuration for this deployment no longer exists.' : 'This ' + (kind | humanizeKind ) + ' is missing.'}}"
```

Comment:
Same comment as above, if the page you are viewing is for something that does not exist, thats a bigger deal than a little warning icon next to the name.